### PR TITLE
573 Python Image

### PR DESCRIPTION
**Pull request by ZacWarham**
I built this based on [my modifications](https://github.com/joelostblom/prairielearn-dockers/pull/1) to @joelostblom [py-r image](https://github.com/UBC-MDS/prairielearn-dockers/pull/15) using the YAML file from the [573 instructor repo](https://github.ubc.ca/mds-2024-25/DSCI_573_feat-model-select_instructors/blob/master/environment.yaml)

It has been deployed as `ubcmds/573-python:67e447e` and can be tested on the live server [here](https://us.prairielearn.com/pl/course/8777/question/9297343/preview)

Please test it out and let me know if everything is working

### CODE REVIEW

**Comment by ZacWarham** [comment]
Just realised I made commits to this branch by mistake for https://github.com/UBC-MDS/prairielearn-dockers/pull/17/commits/a0e2bad94569efa220f9d5cb2147ff511a53ff61 https://github.com/UBC-MDS/prairielearn-dockers/pull/17/commits/6acbbd24fd575c4a3fc3ccf256506bed818c36eb https://github.com/UBC-MDS/prairielearn-dockers/pull/17/commits/dc51f2a2764db0ba4bfab2a23756229518c25be2 and https://github.com/UBC-MDS/prairielearn-dockers/pull/17/commits/1c1d54d43be133856c63c9bc92d35e62b3fb5f71

Please ignore them for this review 😅 - the relevant changes are https://github.com/UBC-MDS/prairielearn-dockers/pull/17/commits/d243a59b1fc5c90b4baef6bc4b6ade897c9288e9

File: 573/python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+USER root
+# Install dependencies and various libraries
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Suppress the opt-in dialog for announcements.  
+# https://stackoverflow.com/questions/75511508/how-to-stop-this-message-would-you-like-to-receive-official-jupyter-news
+RUN jupyter labextension disable @jupyterlab/apputils-extension:announcements
+RUN mamba install --yes \
+    'altair-all' \
```
Comment (joelostblom) [inline 573/python/Dockerfile:24]:
```suggestion
    'altair-all==5.4.1' \
    'vegafusion==1.6.9' \
```

Comment (joelostblom) [reply 573/python/Dockerfile:24]: There were some recent updates that creates a dependency conflict, so let set these specifically for now

**Review by joelostblom — CHANGES_REQUESTED** [review]

**Comment by ZacWarham** [comment]
I have pushed those changes to the new image `ubcmds/573-python:67e447e` and updated the [test question on PL](https://us.prairielearn.com/pl/course/8777/question/9297343/preview)

**Comment by VincentLiu3** [comment]
I tested the image for 573 workspace and it worked. Thank you @ZacWarham!

**Comment by joelostblom** [comment]
I'll go ahead and merge this since it seems to work well, thank Zac!

**Review by joelostblom — APPROVED** [review]

**Comment by ZacWarham** [comment]
Cheers Joel 👌

**Pull request merged**